app/partials/export-history.pug

```diff
@@ -2,7 +2,7 @@ form(role="form" name="exportForm" ng-submit="submit()" autocomplete="off" noval
   .modal-header
     .flex-center.pam
       h3.man.mrm(translate="EXPORT_HISTORY")
-      helper-button(content="EXPORT_HISTORY_EXPLAIN")
+      helper-button.show(content="EXPORT_HISTORY_EXPLAIN")
```

Comment:
Helper buttons are normally hidden at mobile width, but I guess we don't want that to be the case for this one.